bindings/L2ToL1MessagePasserExecuter.go

```diff
@@ -45,7 +46,11 @@ func (e *L2ToL1MessagePasserExecuter) InitiateWithdrawal(
 		return fmt.Errorf("create initiateWithdrawal data: %v", err)
 	}
 
-	senderEthAddress := common.HexToAddress(sender)
+	senderCosmosAddress, err := sdk.AccAddressFromBech32(sender)
+	if err != nil {
+		return fmt.Errorf("convert sender to cosmos address: %v", err)
+	}
```

Comment:
~Can we pass in the `sender` as an SDK address type instead of string?~

Actually, can we make it the eth sender?